ewms_pilot/pilot.py

```diff
@@ -27,7 +27,7 @@
 
 _DEFAULT_TIMEOUT_INCOMING = 1  # second
 _DEFAULT_TIMEOUT_OUTGOING = 1  # second
-_DEFAULT_PREFETCH = 1
+_DEFAULT_PREFETCH = 0
```

Comment:
Is there still a need for this variable, with the change below to `ENV.EWMS_PILOT_PREFETCH`?

Reply:
Nope :smile: I guess flake8 didn't look at this scenario

Reply:
ah, it was used in one other place

Reply:
https://github.com/Observation-Management-Service/ewms-pilot/pull/42